assets/chat/js/chat.js

```diff
@@ -482,7 +487,7 @@ class Chat {
     const onresize = () => {
       // If this is a mobile screen, don't close menus.
       // The virtual keyboard triggers a 'resize' event, and menus shouldn't be closed whenever the virtual keyboard is opened
-      if (window.screen.width <= 768) {
+      if (this.shouldFocus) {
```

Comment:
This check was added because the virtual keyboard triggers a resize event when it appears on Android. In the callback, all menus are closed. This resulted in menus that contain an autofocused input immediately closing after they were opened.

Given we no longer autofocus menu inputs on mobile, I think it's safe to remove this check.